tests/playwright-test/web-server.spec.ts

```diff
@@ -35,7 +35,7 @@ test('should create a server', async ({ runInlineTest }, { workerIndex }) => {
     'playwright.config.ts': `
       module.exports = {
         webServer: {
-          command: 'node ${JSON.stringify(SIMPLE_SERVER_PATH)} ${port}',
+          command: 'node ${JSON.stringify(SIMPLE_SERVER_PATH)} ${port} ${301}',
```

Comment:
I don't understand why 301 here.

Reply:
Ah, I don't need this, left it by accident